rbac/internal/views.py

```diff
@@ -697,6 +697,82 @@ def reset_imported_tenants(request: HttpRequest) -> HttpResponse:
     return HttpResponse("Invalid method", status=405)
 
 
+ALLOWED_ROLE_UPDATE_ATTRIBUTES = {"system", "platform_default", "admin_default"}
+
+
+def str_to_bool(value: str) -> bool:
+    """Convert string to bool."""
+    return value.lower() == "true"
```

Comment:
nitpick: can you trim the empty characters too? `return value.strip().lower() == "true"`